dashboards/cloudsql/1p-cloudsql-general.json

```diff
@@ -0,0 +1,320 @@
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": [
+                        "metric.label.\"state\""
+                      ],
+                      "perSeriesAligner": "ALIGN_MEAN"
+                    }
+                  }
+                }
+              }
+            ],
+            "timeshiftDuration": "0s",
+            "yAxis": {
+              "label": "y1Axis",
+              "scale": "LINEAR"
+            }
+          }
+        },
+        "width": 6,
+        "xPos": 6,
+        "yPos": 2
+      },
+      {
+        "height": 4,
+        "widget": {
+          "title": "Instance States",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "minAlignmentPeriod": "60s",
+                "plotType": "STACKED_AREA",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_NONE",
+                      "perSeriesAligner": "ALIGN_COUNT_TRUE"
+                    },
+                    "filter": "metric.type=\"cloudsql.googleapis.com/database/instance_state\" resource.type=\"cloudsql_database\"",
+                    "secondaryAggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": [
+                        "metric.label.\"state\""
+                      ],
+                      "perSeriesAligner": "ALIGN_MEAN"
+                    }
+                  }
+                }
+              }
+            ],
+            "timeshiftDuration": "0s",
+            "yAxis": {
+              "label": "y1Axis",
+              "scale": "LINEAR"
+            }
+          }
+        },
+        "width": 6,
+        "xPos": 0,
+        "yPos": 2
+      },
+      {
+        "height": 4,
+        "widget": {
+          "logsPanel": {
+            "filter": "resource.type=\"cloudsql_database\"",
+            "resourceNames": []
+          },
+          "title": "All - CloudSQL Logs"
+        },
+        "width": 4,
+        "xPos": 4,
+        "yPos": 8
+      },
+      {
+        "height": 4,
+        "widget": {
+          "logsPanel": {
+            "filter": "resource.type=\"cloudsql_database\"\nlogName=~\".*cloudsql.googleapis.com.*\"\nseverity>=ERROR",
+            "resourceNames": []
+          },
+          "title": "All - CloudSQL Database Error Logs"
+        },
+        "width": 4,
+        "xPos": 8,
+        "yPos": 8
+      },
+      {
+        "height": 4,
+        "widget": {
+          "alertChart": {
+            "name": "<insert alert name here or remove chart>"
+          }
+        },
+        "width": 4,
+        "xPos": 0,
+        "yPos": 14
+      },
```

Comment:
We can't currently support alert charts in sample dashboards in a way that is useful to users, so let's remove these for now. The charts depend on the existence of a project-specific alert instance, and so will appear as an empty chart when imported.

Reply:
removed them for now.

Reply:
Thank you!

Reply:
Thank you. sorry for chime in late. Please remove the alert chart as we don't yet have support for it in sample dash.